.golangci.yml

```diff
@@ -2,7 +2,28 @@ run:
   timeout: 10m
   modules-download-mode: vendor
 linters:
+  disable-all: true
   enable:
-  - whitespace
+    - bodyclose
+    - deadcode
+    - dupl
+    - errcheck
+    - gofmt
+    - goimports
+    - gosimple
+    - govet
+    - misspell
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unused
+    - varcheck
+    - whitespace
```

Comment:
It seems to ignore the [ineffassign](https://golangci-lint.run/usage/linters/#enabled-by-default) that are enabled by default

Reply:
https://golangci-lint.run/usage/linters/ 
yes , I missed it

Reply:
done